src/platforms/unity/migration.mdx

```diff
@@ -10,6 +10,10 @@ redirect_from:
 
 The Sentry SDK deprecated the use of a JSON file to store the options in favor of scriptable objects. If you're migrating from version 0.3.0 or older, make sure to upgrade to version 0.15.0 first, as it is the last version supporting your options stored as JSON. When you're opening the Sentry configuration editor window, that file will be automatically converted into a scriptable object. This support will be dropped in version 0.16.0 and going forward.
 
+## Migrating From 0.7.0 or earlier
+
+Starting from 0.7.0 the SDK is [aliasing](https://blog.sentry.io/2022/02/24/alias-an-approach-to-net-assembly-conflict-resolution) its dependencies. We do this to avoid creating conflicts with other packages. This also means, that if you were relying on our dependencies for certain types, then you will have to import them into your project yourself going forward.
```

Comment:
```suggestion
Starting from version 0.7.0, the SDK is [aliasing](https://blog.sentry.io/2022/02/24/alias-an-approach-to-net-assembly-conflict-resolution) its dependencies. We do this to avoid creating conflicts with other packages. This also means that if you were relying on our dependencies for certain types, then you will have to import them into your project yourself going forward.
```